src/game_api/script/lua_backend.hpp

```diff
@@ -288,6 +290,7 @@ class LuaBackend
 
     void clear();
     void clear_all_callbacks();
+    bool frame_update(StateMemory* state_mem);
```

Comment:
Do we really gain anything from this rename?

Reply:
Not really, originally i left the update so i had to change this to something else

Also didn't want the confusion with the old way where the updated had to be called non-stop where now it's all based on hook